src/Utilities/operate.jl

```diff
@@ -991,7 +991,11 @@ function operate(
             append!(out, scalarize(a))
```

Comment:
You don't need to copy if it's the output of `scalarize`, I would rather copy in the two `push!` above

Reply:
> You don't need to copy if it's the output of scalarize

Ah but you do if you hit this method `MOI.Utilities.scalarize(f::MOI.VectorNonlinearFunction) = f.rows`.

The question of whether `scalarize` and `vectorize` are mutable hasn't really come up before, because they always produced unique objects.

Reply:
:fearful: